custom-recipes/named-entity-recognition-extract/recipe.py

```diff
@@ -51,9 +52,9 @@
 
 def compute_entities_df(df):
     if ner_model == "spacy":
-        out_df = extract_entities(df[text_column_name].fillna(" "), format=output_single_json, language=language)
+        out_df = extract_entities(df[text_column_name].fillna(" "), format=output_json_format if output_single_json else None, language=language)
     else:
-        out_df = extract_entities(df[text_column_name].fillna(" "), format=output_single_json, tagger=tagger)
+        out_df = extract_entities(df[text_column_name].fillna(" "), format=output_json_format if output_single_json else None, tagger=tagger)
```

Comment:
Not a big deal since there is not much code but I think that it is confusing to add an additional layer of parameter-related logic here : to understand the code I need to read 3 places: the 1st common recipe config parsing (around l. 30), this conditional duplicated param assignment (l. 55 & 57) & the extraction method retrieval logic which relies on a smart dict.get behaviour (cf my comment below).

I suggest to: 
* create an `output_format` variable while parsing the recipe config (around line 30) which can take 3 values: `default` , `json_standard`, `json_labeling` (they could be defined as constants reused within both model implementations)
```
output_format = "default"
if advanced_settings:
    use_json = recipe_config.get("output_single_json", False)
    if use_json:
        json_format = recipe_config.get("output_json_format", "standard")  # backward compatibility
        output_format = "json_" + json_format
    ner_model = recipe_config.get("ner_model", "spacy")
else:
    ner_model = "spacy"
```
*  pass through this `output_format` variable directly to the `extract_entities` call
* make the code clearer and stricter within both `extract_entities` methods by declaring a complete dict: 
```
extraction_methods = {
        "json_standard": get_standard_rows,
        "json_labeling": get_labeling_rows,
        "default": get_default_rows
    }
extraction_method = extraction_methods.get(output_format)
rows = extraction_method.(sentences)
```

Reply:
makes sense, I added the three constants :+1: